yaml/com/ibm/VPD/FRUManager.interface.yaml

```diff
@@ -0,0 +1,139 @@
+description: >
+    Implement to manage VPD in the system.
+
+methods:
+    - name: WriteKeyword
+      description: >
+          Method to update a single keyword's value based on the input
+          parameters.
+      parameters:
+          - name: path
+            type: string
+            description: >
+                Path where the data resides(D-Bus inventory path/EEPROM path
+                etc).
+          - name: paramsToWriteData
+            type:
+                variant[struct[string,string,array[byte]],
+                struct[string,array[byte]]]
+            description: >
+                Parameters in required format to write the keyword data.
+      returns:
+          - name: bytesUpdated
+            type: ssize
+            description: >
+                On success, returns the number of bytes updated. On failure,
+                returns -1.
+      errors:
+          - xyz.openbmc_project.Common.Error.InvalidArgument
+          - xyz.openbmc_project.Common.Error.InternalFailure
+
+    - name: ReadKeyword
+      description: >
+          Method to read a single keyword's value based on the input parameters.
+      parameters:
+          - name: path
+            type: string
+            description: >
+                Path where the data resides(D-Bus inventory path/EEPROM path
+                etc).
+          - name: paramsToReadData
+            type: variant[struct[string,string],struct[string]]
+            description: >
+                Parameters in required format to read the keyword data.
+      returns:
+          - name: keywordValue
+            type: variant[array[byte]]
+            description: >
+                On success, returns the Keyword's value read from the given
```

Comment:
nit: On success, returns the keyword's value. On failure, throws an exception.

Reply:
updated